thyrel-api/thyrel-api/Json/WebsocketEventJson.cs

```diff
@@ -89,4 +89,32 @@ public class PlayerFinishStepWebsocketEventJson : BaseWebsocketEventJson
             };
         }
     }
+
+    public class EmojiReactionWebSocketEventJson : BaseWebsocketEventJson
+    {
+        public int PlayerId;
+        public int ElementId;
+        public EmojiReaction EmojiReaction;
+
+        public EmojiReactionWebSocketEventJson(int playerId, int elementId, EmojiReaction emojiReaction)
+        {
+            PlayerId = playerId;
+            ElementId = elementId;
+            EmojiReaction = emojiReaction;
+            WebsocketEvent = WebsocketEvent.NewReactionToElement;
```

Comment:
```suggestion
            WebsocketEvent = WebsocketEvent.NewReaction
```
Y'a pas de reaction sur autre chose que des elements